### DISCUSSION TITLE
Cronologia dati sparita per qualsiasi entità

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#3032] [HUB: Home Assistant]
Salve a tutti, 

Da quando ho aggiornato alla versione 2024.2, è sparito lo storico di qualsiasi entità che ho in HA. Parliamo dello storico delle temperature degli shelly, dello storico del pannello energia, dello storico degli stati dell'interruttore, ecc. ecc. 

Ad esempio, il mio pannello energia è così da alcune settimane:

![image](https://github.com/indomus/forum/assets/60674280/bd91a97e-48a8-4c1a-980f-1c65eecf3afb)

ma anche qualsiasi entità nello storico risulta vuota:

![image](https://github.com/indomus/forum/assets/60674280/67b9184d-357a-46af-b2e0-369f16b4516c)

Avete qualche consiglio da darmi?

Attualmente dispongo di HA core  2024.4 istanziata su Docker, con Rasp3B e SSD esterno.

Fatemi sapere se servono altre info.
Grazie

**Comment by calcioscacchi** [comment]
Un aggiornamento: nella configurazione del pannello energia sembra che il recorder (che però non ho mai configurato) compare quest oerrore:

```
Entità non tracciata
Il registratore è stato configurato per escludere queste entità configurate: ([Ulteriori informazioni](https://www.home-assistant.io/integrations/recorder#configure-filter))
sensor.shellyem_3494547456f6_channel_1_energy
sensor.shellyem_3494547456f6_channel_1_energy_returned
sensor.shellyem_3494547456f6_channel_1_energy_cost
```

Immagino voglia dire che il recorder ha raggiunto il limite? 
Avete dei suggerimenti per aiutarmi a risolvere la situazione? Grazie.

**Reply by Scialla** [reply]
il log dice che hai sbagliato a configurare il recorder e hai escluso l'entità.... controlla la tua configurazione del recorder e togli quelle entità dalle esclusioni, poi riavvia HA
